From 62588f2d8267c67c2337004e83e80695280db9eb Mon Sep 17 00:00:00 2001
From: James Grant <jamesg@zaltys.org>
Date: Sun, 3 Mar 2019 17:29:29 +1300
Subject: [PATCH 1/1] libressl - add -fpermissive gcc flag to allow
 compilation.

Workaround for 'const BIO_METHOD *' vs. 'BIO_METHOD *' changes to BIO_new() and BIO_s_mem() function signatures.

Signed-off-by: James Grant <jamesg@zaltys.org>
---
 src/network/ssl/ssl.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/ssl/ssl.pri b/src/network/ssl/ssl.pri
index 29c47cd7..f0aad448 100644
--- a/src/network/ssl/ssl.pri
+++ b/src/network/ssl/ssl.pri
@@ -77,7 +77,7 @@ android:!android-no-sdk: SOURCES += ssl/qsslsocket_openssl_android.cpp
         LIBS_PRIVATE += $$OPENSSL_LIBS_RELEASE
     }
 
-    QMAKE_CXXFLAGS += $$OPENSSL_CFLAGS
+    QMAKE_CXXFLAGS += $$OPENSSL_CFLAGS -fpermissive
     LIBS_PRIVATE += $$OPENSSL_LIBS
     win32: LIBS_PRIVATE += -lcrypt32
 }
-- 
2.18.1

